src/app.js

```diff
@@ -21,7 +20,10 @@ const httpsServer = https.createServer(credentials, app);
 
 app.use(cors());
 const RTC_CONFIG = JSON.parse(process.env.RTC_CONFIG) || null;
-const SIGNAL_SERVER_URL = process.env.SIGNAL_SERVER_URL;
+// const SIGNAL_SERVER_URL = process.env.SIGNAL_SERVER_URL;
+
+const SIGNAL_SERVER_URL =
+  'wss://rufdlv7k6k.execute-api.us-east-2.amazonaws.com/production';
```

Comment:
this should probably be refactored to an environment variable

Reply:
Oh absolutely. Definitely forgot to put it in environment since I was swapping it a lot for testing